pkg/deploy/gateway/traefik_config_util_test.go

```diff
@@ -0,0 +1,84 @@
+package gateway
```

Comment:
Add license.
I encourage to use asserts from the `github.com/stretchr/testify/assert` package for new tests.

Reply:
the assert package is really nice. fixed